### DISCUSSION TITLE
does Redash work with baserow?

### DISCUSSION THREAD

**Discussion by samuellhwarren** [getredash/redash#6739] [Q&A] [answered]
I am looking to create a database and reporting platform on my linux server. I like the idea of baserow because of it's useability. Can I use redash with baserow? also is there a way to start it behind nginx proxy manager?

**Comment by guidopetri** [comment, marked as answer]
We don't have a query runner for baserow, no, but would gladly accept a PR if you're willing to contribute!

As for your second question, redash can be run behind a reverse proxy, yes.
